chapters/annotations.tex

```diff
@@ -310,10 +303,10 @@ \section{Annotations for Graphical Objects}\label{annotations-for-graphical-obje
 
 \begin{example}
 \begin{lstlisting}[language=modelica]
-annotation (
-   Icon(coordinateSystem(extent={{-100,-100}, {100,100}}),
-        graphics={Rectangle(extent={{-100,-100}, {100,100}}),
-                  Text(extent={{-100,-100}, {100,100}}, textString="Icon")}));
+annotation(
+   Icon(coordinateSystem(extent = {{-100, -100}, {100, 100}}),
+        graphics = {Rectangle(extent = {{-100, -100}, {100, 100}}),
+                    Text(extent = {{-100, -100}, {100, 100}}, textString = "Icon")}));
```

Comment:
For pdf this changes causes a line-break in the string; so put entire textString on the next line.

Reply:
Fixed.  Resolving.